File: ThreadsApp/ThreadsApp/View/Chat/Cells/Label/LabelCell.m
//
//  LabelCell.m
//  ThreadsApp
//
//  Created by Tabriz Dzhavadov on 13/07/2017.
//  Copyright © 2017 Brooma Service. All rights reserved.
//

#import "LabelCell.h"

@interface LabelCell()

@property (weak, nonatomic) IBOutlet UILabel *label;

@end

@implementation LabelCell

#pragma mark - Initialize

- (void)awakeFromNib {
    [super awakeFromNib];
    self.selectionStyle = UITableViewCellSelectionStyleNone;
}

#pragma mark - Configuring

- (void) construct {
    self.label.text = [CellTypeHelper textByType: self.type];
}

@end

File: ThreadsApp/ThreadsAppSwift/View/Chat/Cells/LabelCell.swift
//
//  LabelCell.swift
//  THRAppSwift-Polarbear
//
//  Created by Brooma Service on 16/05/2019.
//  Copyright © 2019 Brooma Service. All rights reserved.
//

import Foundation
import UIKit

class LabelCell : UITableViewCell {
    
    var type: CellType = .Version
    
    @IBOutlet weak var label: UILabel!
    
    // MARK: - Initialize
    
    override func awakeFromNib() {
        super.awakeFromNib()
        
        self.selectionStyle = .none
    }
    
    // MARK: - Configuring
    
    func construct() {
        self.label.text = CellTypeHelper.getCellText(forType: self.type);
        
    }
}
